adblock-lean

```diff
@@ -33,6 +31,9 @@ else
 fi
 
 PREFIX=/root/adblock-lean
+abl_dir=/var/run/adblock-lean
+tmp_abl_dir=/tmp/adblock-lean
```

Comment:
Initially the idea was to use this directory for the lock file. Storing the lock file in `/var/run/adblock-lean` intuitively seems wrong because (for example) if the lock file was stored in the same directory and we had 2 instances of adblock-lean running (for example one is downloading and applying the blocklist, another one is called with the status command), a subtle mistake in the code could lead to race conditions, for example one instance could delete the directory and the lock file along with it while the other instance expects the lock file to be present. It is possible to keep this in mind when coding and carefully test for such conditions, but much easier to just keep a separate directory for the lock file.

Apparently later I got confused and used that directory in the write_config() function. This is a mistake, I'll fix this in a moment.